test_data/gbif/init_test_data.sql

```diff
@@ -0,0 +1,6 @@
+CREATE EXTENSION IF NOT EXISTS postgis WITH SCHEMA public;
+
+SET ROLE 'geoengine'; -- noqa: PRS
```

Comment:
what does the comment mean?

Reply:
sqlfluff does not recognize set role xyz, so this mutes the linter